frontend/server/src/test/java/org/pytorch/serve/ModelServerTest.java

```diff
@@ -1068,6 +1068,55 @@ public void testPredictionMemoryError() throws InterruptedException {
     @Test(
             alwaysRun = true,
             dependsOnMethods = {"testPredictionMemoryError"})
```

Comment:
Why this test case depends on "testPredictionMemoryError"?

Reply:
They're not dependent, I believe `dependsOnMethods` is used as a way to set the order in which tests are called